_component_design/loading-indicators.md

```diff
@@ -0,0 +1,205 @@
+usage: >
+  Loading indicators tell the user that their system is actively retrieving data.
+preview-image: preview-images/loading.svg
+resource: true
+status: Stable
+last-modified: 2017-10-24
+---
+
+{% include toc.html %}
+
+## Introduction
+
+<div class="hxRow">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### When to use
+
+Use loading indicators when you need provide further visual feedback in cases where a system is making large calls or complex operations in the background.
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:""] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/processing/loading-indicator/spinner-use.svg)
+{% endfigure %}
+{% endcolumn %}
+
+</div>
+
+### Best practice
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+- Make sure loading indicators are visible and clearly show the user that information is being retrieved.
+- Center a loading spinner both vertically and horizontally.
+- If loading text is used, always use an ellipses are the word "Loading".
+
+{% endcolumn %}
+
+</div>
+
+## Types of indicators
+
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+Loading indicators can be displayed in the following ways:
+
+### Spinner icon
+
+The spinner icon is a simple circle with a 1 px stroke. It is most commonly used horizontally and vertically centered.
+
+The spinner icon inherits the text color of the element or component in which it is contained.
+
+{% endcolumn %}
+
+{% column right:"hxCol-4 hxCol-xs-6 hxCol-sm-6 hxCol-md-4 hxCol-lg-4" %}
+{% figure [caption:"Spinner icons are centered"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/processing/loading-indicator/spinner-icon-centered.svg)
+
+{% endfigure %}
+{% endcolumn %}
+
+{% column right:"hxCol-4 hxCol-xs-6 hxCol-sm-6 hxCol-md-4 hxCol-lg-4" %}
+{% figure [caption:"The spinner icon uses a 1px stroke width."] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/processing/loading-indicator/spinner-icon-stroke.svg)
+
+{% endfigure %}
+{% endcolumn %}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+A variety of spinner icon sizes are available.
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Spinner icons come in a variety of sizes from small, medium, large, and x-large."] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/processing/loading-indicator/spinner-size.svg)
+
+{% endfigure %}
+{% endcolumn %}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Loading spinner with text
+
+When text is contained inside a spinner icon, the spinner matches the size of the text. Therefore, spinner size is measured in *em*.
```

Comment:
measured in em?!

Reply:
@technabors: We are assuming that designers will know what "em" is as a unit of measurement?

Reply:
Evan?